src/Configuration/ConfigurationValidator.cs

```diff
@@ -50,45 +50,37 @@ public ConfigurationValidator(ILogger<ConfigurationValidator> logger)
         /// <returns></returns>
         public ValidateOptionsResult Validate(string name, DicomAdapterConfiguration options)
         {
-            var valid = IsDicomScpConfigValid(options.Dicom.Scp, options.ReadAeTitlesFromCrd);
-            valid &= IsDicomScuConfigValid(options.Dicom.Scu, options.ReadAeTitlesFromCrd);
+            var valid = IsDicomScpConfigValid(options.Dicom.Scp);
```

Comment:
I assume even if ` IsDicomScpConfigValid(options.Dicom.Scp)` returns false, we want to `IsDicomScuConfigValid(options.Dicom.Scu)` and  `IsServicesValid(options.Services)`

if we just want to fail on the first error
```
var valid = IsDicomScpConfigValid(options.Dicom.Scp)
         && IsDicomScpConfigValid(options.Dicom.Scp)
         && IsServicesValid(options.Services);
```
is more efficient, especially if configuration checking is expensive (no idea if it is).

Reply:
I actually wanted it to go through the entire config file, report all errors so the user can correct them all at once.